code-challenges.js

```diff
@@ -29,40 +45,100 @@ const temp2 = 350
 // Expected output: "350 is above boiling point"
 
 const temp3 = 212
-// Expected output: "212 is at boiling point"
+Expected output: "212 is at boiling point"
+
+
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+// First we will setup the consts as 1 and 2 to give a unique identifer to each array. 
+// I will use a .push method to add the first elements to the second array. 
+// To get the final output of 9 to answer the question I will use console.log combined
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+Expected output: 9
+Given output: 9 
+
+const padres1998WorldSeriesRuns1 = [2, 5, 2, 2, 4]
+const padres1998WorldSeriesRuns2 = [6, 3, 5, 3]
+const combined = padres1998WorldSeriesRuns1.push(...padres1998WorldSeriesRuns2)
+	
+console.log(combined) 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Describe your process:
 
-const currentCohort = "Golf 2022"
+// Describe your process:// Reversing string is one of the most asked JS questions in technical inteviews. 
+// There 3 ways to reverse a string, with built in methods, with a decrementaing for loop, and recusion. Since loop is what was taught lets use the most basic. 
+// Starting with a reverseString funtion  
+// Use a for loop 
+// Console.log will contain a reverseString method to display the 2202 floG in proper sequence and formatting. 
+
+function reverseString(str) {
+    let result = "";
+
+    for (let i = str.length - 1; i >= 0; i -= 1) {
+        result += str[i];
+    }
+    
+    return result;
+}
+    console.log(reverseString("Golf 2022"));
+
+// const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
+// Given output using loop: 2202 floG
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
-
-// Describe your process:
+ 
+// Describe your process: First we have to sort the array and get it indexed. 
+// we assign function and use a, b
+// using a loop i - 0 and then return the arrray in sequence selection the position of 42 and 10. 
+
+function getIndexOfIns(arr, num) {
+	arr.sort( 
+  function(a, b) {
+  	return a - b;
+});
+
+for (var i - 0; i < arr.length; i++ ){
```

Comment:
I am showing this error: "Uncaught SyntaxError: Unexpected token '-'" when running your code and I don't see any comments in your code about running into this error or any attempts to troubleshoot this.  If your code is not working, your pseudocode should discuss what errors you have and what you tried.